src/lambda_toolshed/papillon/channel.cljc

```diff
@@ -0,0 +1,11 @@
+(ns lambda-toolshed.papillon.channel
+  (:require [clojure.core.async :as async]
+            [clojure.core.async.impl.channels :as channels]
+            [lambda-toolshed.papillon.protocols :as protocols]))
+
+(extend-type #?(:clj  clojure.core.async.impl.protocols.Channel
+                :cljs cljs.core.async.impl.channels/ManyToManyChannel)
+  protocols/Chrysalis
+  (emerge
+    #?(:clj ([this] (async/<!! this)))
```

Comment:
I can't remember, is the reader conditional supposed to be in here, and removing the single arity like this for the baseline implementation?

wondering because of the similar snippet in the readme.

Reply:
The blocking/unary arity of emerge is not available in cljs (this is why the async/binary arity came first).  I think the exception thrown is relatively descriptive so I just left it out the implementation.

Reply:
yeah, that's right. sounds good.